Sources/TensorFlow/BackwardsCompatibility.swift

```diff
@@ -615,3 +618,138 @@ public extension Dense {
                   activation: activation)
     }
 }
+
+//===------------------------------------------------------------------------------------------===//
+// Losses
+//===------------------------------------------------------------------------------------------===//
+
+/// Returns the L1 loss between predictions and expectations.
+///
+/// - Parameters:
+///   - predicted: Predicted outputs from a neural network.
+///   - expected: Expected values, i.e. targets, that correspond to the correct output.
+@differentiable(wrt: predicted)
+public func l1Loss<Scalar: TensorFlowFloatingPoint>(
+    predicted: Tensor<Scalar>,
+    expected: Tensor<Scalar>
+) -> Tensor<Scalar> {
+    l1Loss(predicted: predicted, expected: expected, reduction: { $0.sum() })
+}
+
+/// Returns the L2 loss between predictions and expectations.
+///
+/// - Parameters:
+///   - predicted: Predicted outputs from a neural network.
+///   - expected: Expected values, i.e. targets, that correspond to the correct output.
+@differentiable(wrt: predicted)
+public func l2Loss<Scalar: TensorFlowFloatingPoint>(
+    predicted: Tensor<Scalar>,
+    expected: Tensor<Scalar>
+) -> Tensor<Scalar> {
+    l2Loss(predicted: predicted, expected: expected, reduction: { $0.sum() })
+}
+
```

Comment:
Where is `meanAbsoluteError`?

Reply:
@rxwei, these functions are already two parameter functions and don't need to be overloaded.